src/Carousel.js

```diff
@@ -120,17 +120,14 @@ const defaultProps = {
 };
 
 class Carousel extends React.Component {
-  constructor(props, context) {
-    super(props, context);
+  state = {
+    prevClasses: '',
+    currentClasses: 'active',
+  };
 
-    this.state = {
-      prevClasses: '',
-      currentClasses: 'active',
-    };
-    this.isUnmounted = false;
+  isUnmounted = false;
 
-    this.carousel = React.createRef();
-  }
+  carousel = React.createRef();
```

Comment:
not sure if something like this is OK?

Reply:
maybe the assignments could go under cDM? probably doesn't matter too much with `this.isUnmounted`, since it's just a primitive.

Reply:
this is fine